apps/researcher/src/messages/en/messages.json

```diff
@@ -109,14 +109,16 @@
     "next": "Next"
   },
   "Filters": {
-    "ownersFilter": "Owners",
     "typesFilter": "Types",
     "subjectsFilter": "Subjects",
     "birthYearsFilter": "Birth years",
     "birthPlacesFilter": "Birth places",
     "deathYearsFilter": "Death years",
     "deathPlacesFilter": "Death places",
     "publishersFilter": "Object data providers",
+    "materialsFilter": "Materials",
+    "creatorsFilter": "Creators",
+    "locationsFilter": "Locations",
```

Comment:
'Locations' as a label is generic. For example: is this the location of creation, the location where an object currently is, or the location depicted on an object? Is it possible to rename the label and be explicit about its meaning?

Reply:
Now, these names are built by adding 'Filter' to the `SearchResult` filter key. In the `SearchResult`, this key is called `location`. Maybe I could make this name customizable instead of building them depending on the `SearchResult` filter key. Or should the API also return a name that is more explicit about its meaning?

Reply:
1. Good idea to change the key `locations` in the API - I'll write this down
2. Regardless of (1): I think we should not have a tight coupling between the keys that the API uses and values that we present to users. But perhaps I don't understand your comment? We could ask Bas to suggest a value for `locationsFilter`? If `Locations` is the label he wants, I'm fine with that (it's a display issue) :)

Reply:
Sorry, with name. I meant the translation key. In `apps/researcher/src/app/[locale]/page.tsx` this is the code for the `FacetMenu`:

```
function FacetMenu({filters}: FacetMenuProps) {
  const t = useTranslations('Filters');

  return (
      .....
      {facets.map(({key, Component, ...customProps}) => {
       ....
        return (
          <Component
            key={key}
            title={t(`${key}Filter`)}
            testId={`${key}Filter`}
          />
        );
      })}
    ....
  );
}
```

Where title is ` title={t(`${name}Filter`)}`. This key is the same key used in `SearchResult`.

Reply:
I'm sorry; I see now that you were also referring to the actual value and not the translation key. This is a lot easier. I will change the value.